src/popup/compatMode.js

```diff
@@ -1,15 +1,21 @@
+browser.windows.getLastFocused().then((windowInfo) => {
```

Comment:
Is there are reason for using `getLastFocused` over `getCurrent` here?

Reply:
nope, I'll change it, thanks!